README.md

```diff
@@ -1,2 +1,24 @@
 # HW01
-Edit the readme
+## 07/06/2020
+**Hi Everyone**
+
```

Comment:
Something to try in the future would be centering text/images with the following code: <center> and </center> placed above and below whatever you want to center!